core/utils.ts

```diff
@@ -373,11 +374,12 @@ export function parseBlockColour(colour: number|
  *
  * @param fn Function to run.
  * @throws Error Will throw if no global document can be found (e.g., Node.js).
- * @deprecated
+ * @deprecated Use **Blockly.extensions.runAfterPageLoad** instead.
  * @alias Blockly.utils.runAfterPageLoad
  */
 export function runAfterPageLoad(fn: () => void) {
   deprecation.warn(
-      'Blockly.utils.runAfterPageLoad', 'December 2021', 'December 2022');
+      'Blockly.utils.runAfterPageLoad', 'December 2021', 'December 2022',
+      'Blockly.extensions.runAfterPageLoad');
```

Comment:
From https://github.com/google/blockly/pull/5714

> runAfterPageLoad was never intended to be external, but I still left behind a re-export just in case.

So please remove this and set the deprecation message to be something like "No longer provided by Blockly"

Reply:
Fixed!